tests/services/test_feed_validator.py

```
@@ -0,0 +1,172 @@
+async def test_url_with_different_protocols():
+    validator = FeedValidator()
+    url = "https://example.com/feed"
+    with patch("aiohttp.ClientSession.get", new_callable=AsyncMock) as mock_get:
+        mock_get.return_value.__aenter__.return_value.status = 200
+        mock_get.return_value.__aenter__.return_value.text = AsyncMock(return_value="<rss></rss>")
+        is_valid, error = await validator.validate_feed(url)
+        assert is_valid
+        assert error is None
+
+@pytest.mark.asyncio
+async def test_valid_rss_feed():
```

Ruff (0.8.0)

Missing return type annotation for public function `test_valid_rss_feed`

Add return type annotation: `None`

(ANN201)